transactions/set-execution-memory-limit/2022/jun-15/arguments.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+        "type": "Dictionary",
```

Comment:
is this argument correct? I thought we need to pass in something like this:

`[{"type": "UInt64","value": "100000000000.0"}]`